src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -154,6 +191,16 @@ static bool _updateTransform(LottieTransform* transform, float frameNo, bool aut
         _rotateX(&matrix, transform->rotationEx->x(frameNo, exps));
     }
 
+    auto skewAngle = transform->skewAngle(frameNo, exps);
+    // For angles where tangent explodes, the shape degenerates into an infinitely thin line.
+    // This is handled by zeroing out the matrix due to finite numerical precision.
+    if (fabsf(fmod(skewAngle + 90.0f, 180.0f)) < 0.01f) {
```

Comment:
if we don't expect any visual result by this, maybe, we can quickly return false and skip forwarding.